src/pages/billing/List.js

```diff
@@ -67,8 +67,10 @@ const ProjectListItem = ({ project, project: { roles, status }, isActive }) => {
     ])
   }
 
+  const viewerRoles = _.intersection(roles, _.values(billingRoles))
```

Comment:
I am seeing some odd behavior when logged in as Hermione (hermione.owner@test…). Why are no members showing for this billing project? 

![image](https://user-images.githubusercontent.com/484484/161982372-5410b6fe-1273-44aa-90d4-782146a82464.png)

Note also that Hermione is in the alpha spend group, if you want to test that functionality.

Reply:
Also, I would expect to see myself listed in the members. For example, the billing project shown below was owned by Hermione. She made my account a User, so now it shows under "Shared with You". But it doesn't show me as a User in the members list.

![image](https://user-images.githubusercontent.com/484484/161983172-b89a93be-f338-4289-b8f7-0d181d57ecde.png)

Reply:
Regarding the empty Members list, I'm going to guess that it's because that project is orphaned and doesn't have any owners. You can't get into that state from Terra UI, but if a user is extremely determined- they can manipulate the policy directly in Sam to get into this state.

Regarding not seeing yourself- that's another Sam quirk. A `User` on a project can't read the `User` policy so that's why it's not returned by the backend. That said, I can insert the entry manually in the UI code to improve the UX of this.

Reply:
Yes, that's correct that you won't see other members with the `User` role on the billing project. fwiw, this design has already been approved by both product and UX

Reply:
OK, from my conversations with Adam it wasn't clear to me that he was aware. I have posted in Slack about this to make sure we are all on the same page.